pybikes/baksi.py

```diff
@@ -0,0 +1,72 @@
+# -*- coding: utf-8 -*-
+# Copyright (C) 2010-2012, eskerda <eskerda@gmail.com>
+# Distributed under the AGPL license, see LICENSE.txt
+
+import re
+
+from .base import BikeShareSystem, BikeShareStation
+from . import utils
+from contrib import TSTCache
+
+__all__ = ['Baksi', 'BaksiStation']
+
+ID_NAME_RGX="([0-9]+)\-([\w\s.()-]+)\'"
+STATUS_RGX="Durum\ [&nbsp;]+\ (\w+)"
+DOCKS_RGX="Park[&nbsp;]+([0-9]+)"
+BIKES_RGX="Bisiklet[&nbsp;]+([0-9]+)"
+LAT_LNG_RGX="([\s0-9.]+)\',\ \'([\s0-9.]+)"
+
+USERAGENT = "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Ubuntu Chromium/31.0.1650.63 Chrome/31.0.1650.63 Safari/537.36"
+
+cache = TSTCache(delta=60)
```

Comment:
TSTCache is only used for systems where it makes sense to save the previous result (imagine a system where all subsystems request the same URL, we do not want to hit their server 200 times if response is the same). On this case, I think it's not needed, as each network has a different endpoint